repositories/src/main/scala/cromwell/pipeline/datastorage/dto/Project.scala

```diff
@@ -9,6 +9,17 @@ import play.api.libs.json.JsonNaming.SnakeCase
 import play.api.libs.json._
 import slick.lifted.MappedTo
 
+trait Projects {
+  def projectId: ProjectId
+  def ownerId: UserId
+  def name: String
+  def active: Boolean
+  def version: PipelineVersion
+  def visibility: Visibility
+
+  def uniqueNameForProjectVersioning: String = s"$name $ownerId"
```

Comment:
You should add "-" or "_" betwen $name and $ownerId. I Think.

Reply:
That is a very nice point, thanks